### PR TITLE
Add skip to processing modes

### PR DESCRIPTION
**Pull request by aficsor**
Extend the explanation of JPTT processing modes, also updated images.

### CODE REVIEW

File: dashboard2/settings/jptt.md [hunk 1]
```diff
@@ -151,3 +151,12 @@ HTML:
 "%"."exampleObject"."exampleNestedHTMLinJS"."nestedHTML" html
 ```
 **Note:** The JSON Path tester tool is **not equipped** to display the nested use case.
+
+#### Skip
+
+Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule.
```
Comment (Hootyoo) [inline dashboard2/settings/jptt.md:157]: Maybe highlight the `.! ` before `skip`. It's not needed for the others so users may go by their gut feeling and omit it.

Comment (aficsor) [reply dashboard2/settings/jptt.md]: Done.

File: dashboard2/settings/jptt.md [hunk 2]
```diff
@@ -151,3 +151,12 @@ HTML:
 "%"."exampleObject"."exampleNestedHTMLinJS"."nestedHTML" html
 ```
 **Note:** The JSON Path tester tool is **not equipped** to display the nested use case.
+
+#### Skip
+
+Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule. The exclamation mark in the end of the rule is mandatory, so please use the following syntax: `! skip`
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:157]: Is this true, though? Doesn't the `!` actually _disable_ the rule in question without removing it?

File: dashboard2/settings/jptt.md [hunk 3]
```diff
@@ -62,12 +62,12 @@ Click on any of the blue + icons to generate a **JS path** for the string in que
...
-"%"."exampleUrl"
-"%"."exampleHtmlString"
+"%"."exampleUrl" url
+"%"."exampleHtmlString" html
 "%"."exampleObject"."sentence01"
 "%"."exampleObject"."sentence02"
-"%"."exampleObject"."nestedObject"."sentence03"
-"%"."exampleObject"."nestedObject"."sentence04"
+"%"."exampleObject"."nestedObject".*
+"%"."exampleObject"."nestedObject"."sentence04".! skip
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:70]: Isn't this a _disabled rule_?

**Review by zmeggyesi — CHANGES_REQUESTED** [review]
I think we have the `!` and `skip` mixed up. Plus, with this update, the _order_ becomes significant, since the proxy employs a top-to-bottom, break-on-first-match approach, if I'm reading correctly.

File: dashboard2/settings/jptt.md [hunk 4]
```diff
@@ -62,16 +62,16 @@ Click on any of the blue + icons to generate a **JS path** for the string in que
 
 ```
 "%"."exampleVar"
-"%"."exampleUrl" url
+#"%"."exampleUrl" url
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:65]: > If you generate paths for all available strings in the example , the list of paths in the upper text field should look like this

Why? The tester won't add a commented line to the list, will it?

File: dashboard2/settings/jptt.md (hunk 3 above)
Comment (zmeggyesi) [reply dashboard2/settings/jptt.md:70]: This was proven to be false

File: dashboard2/settings/jptt.md [hunk 5]
```diff
@@ -154,9 +154,13 @@ HTML:
 
 #### Skip
 
-Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule. The exclamation mark in the end of the rule is mandatory, so please use the following syntax: `! skip`
+Use this processing mode to mark child node not to be translated. **Order is important!**
+If you use `*` or `**` to select child nodes to be translated, you can use the `! skip` processing mode to select a child element as non-translatable, but please note, that skip rules must come after the generic `*` rule. Skip is the lowest level mode, but you can increase the priority by adding `!` or `!{number}`
+You can also comment out elements by adding `#` at the start of a line. In this case the parser won' process the given rule at all.
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:159]: Typo here, missing a `t` in `won't`.

File: dashboard2/settings/jptt.md [hunk 6]
```diff
@@ -154,9 +154,13 @@ HTML:
 
 #### Skip
 
-Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule. The exclamation mark in the end of the rule is mandatory, so please use the following syntax: `! skip`
+Use this processing mode to mark child node not to be translated. **Order is important!**
+If you use `*` or `**` to select child nodes to be translated, you can use the `! skip` processing mode to select a child element as non-translatable, but please note, that skip rules must come after the generic `*` rule. Skip is the lowest level mode, but you can increase the priority by adding `!` or `!{number}`
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:158]: While this is true, the `!` switch is not only for `skip` rules, technically. Any rule can be appended to increase its priority above other matching rules, no?

Perhaps the `!` could be a separate heading on its own, along with commenting lines?

**Review by zmeggyesi — CHANGES_REQUESTED** [review]

**Review by zmeggyesi — APPROVED** [review]
LGTM!

**Pull request merged**